station/config.yml.template

```diff
@@ -25,5 +25,5 @@ satellites:
 server:
   id: '1'
   secret: 0123456789abcdef0123456789abcdef
-  url: http://127.0.0.1:5000/receive
+  url: https://svarog.space/receive
```

Comment:
Are you sure that it should be our server by default? It is easier for us, but new users probably will be test software locally first.

Reply:
We're not exactly having a problem with too many users right now. I'd open a bottle of champagne if there's anyone except us submitting any observations. Ok, we could keep it as 127.0.0.1:5000, but the proper URL should be there commented out. Would that work?